### PR TITLE
Remove `REFUNDED` and `COMPLETED` from `lib/constants/expense-status.js`

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/5582

image

### CODE REVIEW

File: components/transactions/TransactionItem.js [hunk 1]
```diff
@@ -108,7 +108,7 @@ const getExpenseStatusTag = (expense, isRefund, isRefunded) => {
   if (isRefunded) {
     expenseStatusLabel = expenseStatus.REFUNDED;
```
Comment (Betree) [inline components/transactions/TransactionItem.js:109]: Isn't this now `undefined`?

Comment (Betree) [reply components/transactions/TransactionItem.js:109]: Can you double-check and test?

image

image

Comment (SudharakaP) [reply components/transactions/TransactionItem.js]: @Betree : Ops, I am so sorry, I somehow got confused here, yes you are right, there's an error on using a non-existent constant here. I've corrected it. 👍🏽

**Review by Betree — APPROVED** [review]
Looks good!

**Pull request merged**